FlagsmithClient/src/main/java/com/flagsmith/builder/Flagsmith.kt

```diff
@@ -0,0 +1,192 @@
+package com.flagsmith.builder
```

Comment:
It looks like this file contains the Flagsmith client itself - is there a reason it's stored in a `builders` package?

Reply:
Now flagsmith